wherehows-web/mirage/factories/lineage-view.ts

```diff
@@ -0,0 +1,32 @@
+import { Factory, faker } from 'ember-cli-mirage';
+import { DatasetPlatform } from 'wherehows-web/constants';
+import { hdfsUrn, nonHdfsUrn } from 'wherehows-web/mirage/fixtures/urn';
+
+export default Factory.extend({
+  dataset() {
```

Comment:
This isn't DRY, the `datasetView` [factory](
      WhereHows/wherehows-web/mirage/factories/dataset-view.ts
    ) already specifies a dataset schema and should be reused. This augmentation should be done in the mirage route handler instead defined [here](
      WhereHows/wherehows-web/mirage/config.ts
    ). If in the future, the `datasetView` interface changes, then you / someone else will have to go in and update all the definitions rather than just updating it in one place.
Let me know if this is unclear?